### PR TITLE
Mini langstream

### PR DESCRIPTION
**Pull request by nicoloboschi**
* Added `mini-langstream` script with instructions. By default it installs the latest version of langstream.
It requires:
* Minikube
* kubectl
* helm
* docker
* langstream (cli)

When you run `mini-langstream start`, the following happens:
* Minikube context is setup
* MinIO is created inside a docker container
* Kafka is created inside a docker container
* LangStream is deployed using helm in the minikube context
* PortForwarding is enabled for gateway and control plane
* A dedicated CLI config is created

You can then use the CLI targeting the cluster with:
```
mini-langstream cli apps list
```
* Added to the release procedure
* Added binary installation similar to langstream CLI

### CODE REVIEW

File: bin/mini-langstream/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# mini-langstream
+
+## Get started
+
+### MacOS/Linux/WSL
+
+```
+curl -Ls "https://raw.githubusercontent.com/LangStream/langstream/main/bin/mini-langstream/get-mini-langstream.sh" | bash
```
Comment (eolivelli) [inline bin/mini-langstream/README.md:8]: Can we make it available in the homebrew formula ?

Personally (maybe it is a matter of taste) I don't like to install stuff this way on my Mac, as there is no way of uninstalling things. With homebrew (and other managers) the whole lifecycle is handled (also the upgrades)

File: bin/mini-langstream/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# mini-langstream
```
Comment (eolivelli) [inline bin/mini-langstream/README.md:1]: I am not sure that the README should go into the "bin" directory,
maybe it is better to move the whole directory to the root (not as child of "bin")

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: bin/mini-langstream/README.md (hunk 1 above)
Comment (nicoloboschi) [reply bin/mini-langstream/README.md:8]: yes this is for Linux users. Brew will come later

File: bin/mini-langstream/README.md (hunk 2 above)
Comment (nicoloboschi) [reply bin/mini-langstream/README.md:1]: good point

**Review by eolivelli — APPROVED** [review]

**Pull request merged**